groups/alteryx-sandbox/data.tf

```diff
@@ -49,7 +49,7 @@ data "aws_subnet_ids" "automation" {
 
 data "aws_subnet" "automation" {
   for_each = data.aws_subnet_ids.automation.ids
-  id = each.value
+  id       = each.value
```

Comment:
Resource arguments should be separated from meta arguments, like `for_each` with a whitespace.